tools/keytools/sign.c

```diff
@@ -70,6 +70,7 @@
 #endif
 #ifdef HAVE_ECC
     #include <wolfssl/wolfcrypt/ecc.h>
+    #define MAX_ECC_KEY_SIZE 66
```

Comment:
Why not use `ECC_MAXSIZE`?

Reply:
I copied this from keygen.c, will replace both